src/server.js

```diff
@@ -48,6 +50,50 @@ module.exports = function() {
 			maxAge: 86400 * 1000,
 		}));
 
+	app.get("/uploads/:file", (req, res) => {
```

Comment:
Perhaps we should move this handler to `uploads.js`.

Reply:
package and theme serving also lives here, fwiw.

Reply:
I know, but we should move it out at some point.